### ISSUE TITLE
[Question] : Can i get Locator object from an Element Handle object ?

### ISSUE THREAD

**Issue by GunjanSheth** [microsoft/playwright#17336]
Scenario : 
I have a locator instance that is representing a Web Table on a page.
lets say 
`table_locator = page.locator("xpath=table")`

I have another function that would return me a list of rows in this table

def get_rows():
    return table_locator.locator("xpath=./tbody/tr")
    
now I want to make another function that would search for a text in specific columns in each row.

```
def find_row_with_text(text_to_match, column_index):
    # iterate through all the rows
    for single_row in get_rows().element_handles():        # if I dont use element_handles() I cannot iterate
        # row.locator("xpath=.//td")  . . .  this does not work, so have to go with the following
        # row.query_selector("xpath=.//td").nth(column_index)  . . . does not work
        cell = row.query_selector(f"xpath=./td[{column_index}]")    # this will work but here I had to change the Xpath just as a Work Around
```
        
        
**_So from Locator  -> i got a list of ElementHandles -> then for each ElementHandle object I want to get a specific index. How would I get that ?_**

The workaround is use a different query_selector, but that looks strange.

**Comment by mxschmitt** [comment]
We discourage from using query_selector. Is it only about the looping? You can loop over Locators as well, see here: https://playwright.dev/python/docs/locators#lists

**Comment by GunjanSheth** [comment]
yeah i saw that thanks @mxschmitt , but i was looking if I could use the "for .. in " loop.
But i can safely assume its not present right ?
